cmake/module/TryAppendCXXFlags.cmake

```diff
@@ -49,7 +43,7 @@ function(try_append_cxx_flags flags)
     TACXXF                            # prefix
     "SKIP_LINK"                       # options
     "TARGET;VAR;SOURCE;RESULT_VAR"    # one_value_keywords
```

Comment:
It seemed to me that `SOURCE` is also unused here, see: https://github.com/bitcoin/bitcoin/commit/031ac7f366263bacab152aa94a51025c582b1191#diff-d7f6f763fed50046f1c8a5dcca088fabb6b8c3662a55516f08fc0d0ba24829e3L45